packages/psss/src/containers/ReportsSubmittedCard.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Typography from '@material-ui/core/Typography';
+import { CircleMeter, Card, CardContent, CardHeader } from '@tupaia/ui-components';
+import Skeleton from '@material-ui/lab/Skeleton';
+import { useReportsSubmitted } from '../api/queries';
+import { getEntitiesAllowed } from '../store';
+import { connect } from 'react-redux';
+
+const StyledCardContent = styled(CardContent)`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+export const ReportsSubmittedCardComponent = React.memo(({ countryCodes }) => {
```

Comment:
As above, perhaps this component should be renamed to be a bit more in line with the fact that it displays the confirmed countries?